src/natcap/invest/habitat_quality.py

```diff
@@ -2,6 +2,7 @@
 """InVEST Habitat Quality model."""
 import collections
 import logging
+import math
```

Comment:
Is `math` used?

Reply:
`flake8` is also suggesting that `osr` is imported but unused.

Reply:
Thanks, I need to incorporate flake8 back into my workflow

Reply:
Fixed https://github.com/natcap/invest/pull/1159/commits/30ba06614bd613c6870a36a4b8b4a0562cf2454b